apps/src/applab/applab.js

```diff
@@ -666,6 +674,17 @@ Applab.init = function(config) {
     }
     drawDiv();
 
+    // Ignore the user's levelHtml for levels without design mode. levelHtml
+    // should never be present on such levels, however some levels do
+    // have levelHtml stored due to a previous bug. HTML set by levelbuilder
+    // is stored in startHtml, not levelHtml.
+    if (config.level.hideDesignMode) {
+      config.level.levelHtml = '';
+    }
+
+    // #designModeViz doesn't exist until after drawDiv() is called.
```

Comment:
Would there be a benefit to creating an empty version of designModeViz so that we know it always exists?